### ISSUE TITLE
Getting below error when we use from perl 5.30 for Specio 0.47

### ISSUE THREAD

**Issue by vinodarokiya** [houseabsolute/Specio#20]
Can't locate object method "_allow_classes" via package "Specio::Constraint::ObjectCan" at /opt/localperl/lib/site_perl/5.30.0/Specio/Constraint/Role/CanType.pm line 47.

May I plz know if we need to have any other dependency that needed to be installed?

**Comment by autarch** [comment]
Can you provide some code to reproduce this error? I can't do much with just an error message.

**Comment by vinodarokiya** [comment]
Hi Dave,

We have a package defined as Util :

package Util;

use strict;
use warnings;
use DateTime;

sub get_datetime {
  my $dt = DateTime->now;
  my $current_dt = $dt->datetime();
  return $current_dt;
}



In one of the main script we use:
use Util qw(:all);

We are getting Compilation issue while trying to compile Util.pm itself: 
Can't locate object method "_allow_classes" via package "Specio::Constraint::ObjectCan" at ~/perl5.30/Specio/Constraint/Role/CanType.pm line 47, <DATA> line 755.
Compilation failed in require at ~/depot/scm/lib/perl5.30/DateTime/Duration.pm line 12, <DATA> line 755.
BEGIN failed--compilation aborted at ~/depot/scm/lib/perl5.30/DateTime/Duration.pm line 12, <DATA> line 755.
Compilation failed in require at ~/depot/scm/lib/perl5.30/DateTime.pm line 14, <DATA> line 755.
BEGIN failed--compilation aborted at ~/depot/scm/lib/perl5.30/DateTime.pm line 14, <DATA> line 755.
Compilation failed in require at /opt/localperl/lib/site_perl/5.30.0/x86_64-linux/Util.pm line 5, <DATA> line 755.
BEGIN failed--compilation aborted at /opt/localperl/lib/site_perl/5.30.0/x86_64-linux/Util.pm line 5, <DATA> line 755.
Compilation failed in require at sox_es3a.pl line 13, <DATA> line 755.

**Comment by autarch** [comment]
That's very odd. Can you try reinstalling Specio? I really don't know what could be causing this.

**Comment by vinodarokiya** [comment]
Yes i tried installing version 0.47 but still same issue,. I will check if something else is causing this issue even I wonder what s wrong :(

**Comment by autarch** [comment]
How are you installing modules? Are you using `cpanm`? Also, how did you install Perl?

This error suggests to me that there's something weird about your perl install or how you're installing modules.

**Comment by autarch** [comment]
Doing it manually as you described should work fine.

Honestly, I'm really not sure what's going. I know that DateTime and Specio generally just work for people, and I don't think I've seen this error before.

**Comment by vinodarokiya** [comment]
Sure thanks for checking this out,let me check if I can figure out the issue
